client/my-sites/plan-features/index.jsx

```diff
@@ -266,6 +276,10 @@ class PlanFeatures extends Component {
 				}
 			}
 
+			if ( isInSignupTest && displayJetpackPlans ) {
+				billingTimeFrame = planConstantObj.getSignupBillingTimeFrame();
+			}
```

Comment:
It's just different from the test. In a previous conversation with @markryall we discussed keeping logic out of the constants.js file. I would definitely consider this approach in the future. Do you see this being a blocker for the test going out?